dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04210CreateDefaultLanguageVariable.java

```diff
@@ -0,0 +1,183 @@
+package com.dotmarketing.startup.runonce;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static com.dotcms.util.CollectionsUtils.map;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.dotcms.contenttype.model.field.LegacyFieldTypes;
+import com.dotcms.contenttype.model.type.BaseContentType;
+import com.dotcms.contenttype.model.type.KeyValueContentType;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import com.liferay.util.StringPool;
+
+/**
+ * This upgrade task create the default Language Variable Content Type for existing dotCMS installs.
+ * The tables affected by this task are:
+ * <ul>
+ * <li>{@code inode}: Contains the Inodes for the Language Variable and its associated Fields.</li>
+ * <li>{@code field}: Contains the 2 Fields for the Language Variable.</li>
+ * <li>{@code structure}: Contains the new Language Variable Content Type.</li>
+ * </ul>
+ * 
+ * @author Jose Castro
+ * @version 4.2.0
+ * @since Jun 30, 2017
+ *
+ */
+public class Task04210CreateDefaultLanguageVariable implements StartupTask {
+
+    private static final String LANGUAGE_VARIABLE_INODE = "f4d7c1b8-2c88-4071-abf1-a5328977b07d";
+    private static final String LANGUAGE_VARIABLE_VARNAME = "Languagevariable";
+    private static final String LANGUAGE_VARIABLE_NAME = "Language Variable";
+    private static final String LANGUAGE_VARIABLE_DESCRIPTION = "Language Variable as Content";
+    private static final String INSERT_INODE_QUERY = "INSERT INTO inode (inode, owner, idate, type) VALUES (?, ?, ?, ?)";
+    private static final String INSERT_FIELD_QUERY = "INSERT INTO field "
+                    + "(inode, structure_inode, field_name, field_type, field_relation_type, field_contentlet, "
+                    + "required, indexed, listed, velocity_var_name, sort_order, field_values, regex_check, "
+                    + "hint, default_value, fixed, read_only, searchable, unique_, mod_date) "
+                    + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+    private static final String INSERT_STRUCTURE_QUERY =
+                    "INSERT INTO structure (inode, name, description, structuretype, system, fixed, velocity_var_name, host, "
+                                    + "folder, mod_date, default_structure) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
+    private static final String[] LANGUAGE_VARIABLE_FIELD_INODES = {"05b6edc6-6443-4dc7-a884-f029b12e5a0d", // Key
+            "c7829c13-cf47-4a20-9331-85fb314cef8e" // Value
+    };
+
+    @Override
+    public void executeUpgrade() throws DotDataException, DotRuntimeException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)